src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer/EvaluationsTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.Azure.CognitiveServices.Personalizer.Models;
+using Microsoft.Azure.Test.HttpRecorder;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit;
+using System.Collections.Generic;
+using System;
+
+namespace Microsoft.Azure.CognitiveServices.Personalizer.Tests
+{
+    public class EvaluationsTests : BaseTests
+    {
+        [Fact]
+        public void GetEvaluations()
+        {
+            using (MockContext.Start(this.GetType().FullName))
+            {
+                HttpMockServer.Initialize(this.GetType().FullName, "GetEvaluations");
+
```

Comment:
nit: any reason for adding blank lines? If there isn't a particular reason you should remove it.
When styleCop or static analyzers are enabled, you will be forced to fix this as all these warnings will be reported as errors.

Reply:
fixed